packages/noco-docs/content/en/getting-started/installation.md

```diff
@@ -57,6 +57,14 @@ npm install
 npm start
 ```
 
+### Homebrew
+
+```bash
```

Comment:
Replace it with

```bash
```bash
brew tap nocodb/nocodb
brew install nocodb
nocodb
```

Reply:
Updated